docs/tutorials/toy-connector.md

```diff
@@ -265,23 +267,377 @@ def _call_api(endpoint, token):
 
 
 def check(config):
-    # Assert required configuration was provided
-    if "api_key" not in config or "stock_ticker" not in config:
-        log("Input config must contain the properties 'api_key' and 'stock_ticker'")
+    # Validate input configuration by attempting to get the price of the input stock ticker for the previous day
```

Comment:
i made the mistake of running `./gradlew format` which seems to have adjusted all of the blank spacing for the code snippets.